CMakeLists.txt

```diff
@@ -284,8 +295,8 @@ if (NOT ANDROID AND NOT WEBGL)
     add_subdirectory(${EXTERNAL}/glslang/tnt)
     add_subdirectory(${EXTERNAL}/imgui/tnt)
     add_subdirectory(${EXTERNAL}/libassimp/tnt)
-    add_subdirectory(${EXTERNAL}/libpng/tnt)
     add_subdirectory(${EXTERNAL}/libsdl2/tnt)
+    add_subdirectory(${EXTERNAL}/libpng/tnt)
```

Comment:
Why this change?

Reply:
I don't recall doing something around libpng. Maybe merge commits